dcicutils/jh_utils.py

```diff
@@ -149,11 +149,20 @@ def find_valid_file_or_extra_file(obj_id, format):
     home_dir = '/home/jovyan'
     # hardcoded so that we can find which volume to look in by file type
     # will need to be updated if more File types are added to proc file bucket
-    if 'FileVistrack' in file_meta['@type'] or 'FileProcessed' in file_meta['@type']:
+    if file_meta['open_data_url']:
```

Comment:
This will crash if this field isn't present. You need to use `.get('open_data_url', None)`.

Reply:
Updated.